package.json

```diff
@@ -54,6 +55,9 @@
     "mini-css-extract-plugin": "0.8.0",
     "ora": "3.0.0",
     "raf": "3.4.1",
+    "react": "16.9.0",
```

Comment:
`@testing-library/react-hooks` used a feature in `react@^16.9`.
It doesn't affect the hooks themselves but only needs `react` to be over `16.9` in the dev environment.
I'm not very sure if this is a good way to do it.